universal-application-tool-0.0.1/test/services/export/CsvExporterTest.java

```diff
@@ -123,6 +123,10 @@ private void answerQuestion(
         QuestionAnswerer.answerFileQuestion(applicantDataOne, answerPath, "my-file-key");
         // applicant two did not answer this question.
         break;
+      case ID:
+        QuestionAnswerer.answerIDQuestion(applicantDataOne, answerPath, "123");
```

Comment:
Would it be helpful to make give one of these inputs a leading zero so we can verify that it's preserved in the CSV?